apps/src/templates/teacherDashboard/SectionsPage.jsx

```diff
@@ -71,10 +72,22 @@ class SectionsPage extends Component {
     this.props.loadClassroomList();
   };
 
-  handleImportClose = (selectedId) => {
+  handleImportCancel = () => {
     this.setState({rosterDialogOpen: false});
-    // TODO (josh): use `selectedId`.
-    console.log(selectedId);
+  };
+
+  handleImport = courseId => {
+    this.props.importClassroomStarted();
+
+    $.getJSON('/dashboardapi/import_google_classroom', { courseId }).then(() => {
+      this.setState({rosterDialogOpen: false});
+      this.setState({sectionsLoaded: false});
+
+      $.getJSON("/v2/sections/").done(results => {
+        this.props.setSections(results, true);
+        this.setState({sectionsLoaded: true});
+      });
```

Comment:
Both of these APIs set sectionsLoaded. If the call to /v2/sections finishes first, we end up with `sectionsLoaded = true`. If `import_google_classroom` finishes first, we end up with `sectionsLoaded = false`